sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TablesExtensionConfigProvider.cs

```diff
@@ -120,6 +122,15 @@ private async Task<TableClient> CreateTableClient(TableAttribute attribute, Canc
             return table;
         }
 
+        internal Task<ParameterBindingData> CreateParameterBindingData(TableAttribute attribute, ValueBindingContext context)
+        {
+            var tableDetails = new TablesParameterBindingDataContent(attribute);
+            var tableDetailsBinaryData = new BinaryData(tableDetails);
+            var parameterBindingData = new ParameterBindingData("1.0", "AzureStorageTables", tableDetailsBinaryData, "application/json");
```

Comment:
If "application/json" is available in the Mime library, let's use that constant instead, otherwise this is fine

Reply:
I didn't see it available as default. I'll leave this as is